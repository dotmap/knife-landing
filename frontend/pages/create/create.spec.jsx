/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import CreatePage from '.'

describe('About page', () => {
  it('matches snapshot', () => {
    const component = renderer.create(<CreatePage />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
