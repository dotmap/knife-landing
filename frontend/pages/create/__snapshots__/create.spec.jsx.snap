// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page matches snapshot 1`] = `
<main>
  <div
    className="css-hg7ow6"
    width={1}
  >
    <div
      className="css-ubbhdj"
      width={
        Array [
          1,
          null,
          0.6666666666666666,
          0.5,
        ]
      }
    >
      <h2
        className="css-1fou44i"
        fontFamily="sans"
        fontSize={
          Array [
            30,
            50,
          ]
        }
        fontWeight="bold"
      >
        Event creation is coming soon.
      </h2>
      <div
        className="css-40lsg9"
        fontFamily="serif"
      >
        Sign up for our newsletter to stay up-to-date on our progress:
      </div>
      <div
        className="css-431qa7"
      >
        <input
          className="css-19no32u-Input e18zfg970"
          placeholder="awesome.name@your.email"
          type="email"
        />
        <button
          className="css-1jdzszd"
          color="black"
          fontSize="inherit"
          fontWeight="bold"
        >
          Sign Up
        </button>
      </div>
    </div>
  </div>
</main>
`;
